<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OverlayWindow</class>
 <widget class="QWidget" name="OverlayWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>309</width>
    <height>672</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Input Device:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="listWidget_input_device">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Input Connection:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="listWidget_input_connection">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Video Format:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_display_mode_auto_detection">
     <property name="text">
      <string>Auto Format Detection</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="listWidget_display_mode">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_audio">
     <property name="text">
      <string>Audio</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
